File: testLib/src/main/java/me/ycdev/android/lib/test/log/AndroidLogHelper.java
package me.ycdev.android.lib.test.log;

public class AndroidLogHelper {
    // Copy the priority constants from android.util.Log
    public static final int VERBOSE = 2;
    public static final int DEBUG = 3;
    public static final int INFO = 4;
    public static final int WARN = 5;
    public static final int ERROR = 6;
    public static final int ASSERT = 7;

    public static String getPriorityName(int priority) {
        switch (priority) {
            case VERBOSE:
                return "V";
            case DEBUG:
                return "D";
            case INFO:
                return "I";
            case WARN:
                return "W";
            case ERROR:
                return "E";
            case ASSERT:
                return "A";
            default:
                return "U";
        }
    }
}

File: testLib/src/main/java/me/ycdev/android/lib/test/log/AndroidLogHelper.kt
package me.ycdev.android.lib.test.log

@Suppress("MemberVisibilityCanBePrivate")
object AndroidLogHelper {
    // Copy the priority constants from android.util.Log
    const val VERBOSE = 2
    const val DEBUG = 3
    const val INFO = 4
    const val WARN = 5
    const val ERROR = 6
    const val ASSERT = 7

    fun getPriorityName(priority: Int): String {
        return when (priority) {
            VERBOSE -> "V"
            DEBUG -> "D"
            INFO -> "I"
            WARN -> "W"
            ERROR -> "E"
            ASSERT -> "A"
            else -> "U"
        }
    }
}
